src/veins/modules/mobility/traci/TraCIScenarioManager.cc

```diff
@@ -486,6 +489,11 @@ void TraCIScenarioManager::addModule(std::string nodeId, std::string type, std::
 
 	int32_t nodeVectorIndex = nextNodeVectorIndex++;
 
+	// FIXME: vehicle sizes are hardcoded
```

Comment:
Vehicle size still needs to be pulled from a better source (SUMO, probably?)

Reply:
Vehicle lengths are available from SUMO, width as well, though only for visualization. Height is not covered in SUMO AFAIK, so that should probably come from the `omnetpp.ini` with some sane defaults in the ned files.

Reply:
width and height have entries in the [TraCI documentation](http://sumo.dlr.de/wiki/TraCI/Vehicle_Value_Retrieval). All three are listed there without qualification and in the same part of the python interface; even if they're only intended for visualization, I'd say those enough, because we don't have access to the vehicle shapes anyway AFAIK. I added this [here](https://github.com/sommer/veins/pull/35/commits/0f1e88653903bdcad39b4af2650c276dd554e626), and that code executes (but I did not do any reception tests -- see also #35.

Reply:
Ah, indeed. Width and also height (which is not documented in the wiki) are apparently attributes of the [vType](http://sumo.dlr.de/xsd/routeTypes.xsd)